demos/utils.h

```diff
@@ -0,0 +1,21 @@
+/*
+ * Copyright 2019 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *   https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+// Forced memory load.
```

Comment:
Should the comments here be more extensive? If I come to this file without knowing much about the side channel stuff would there be some info I'd want to know here other than this?

Maybe something like:

// Forced memory load. This is typically used to load a byte into cache for Flush+Reload or other cache timing side channels.

Reply:
Done.